code/modules/mob/living/carbon/human/species.dm

```diff
@@ -18,8 +18,7 @@
 
 	var/sexes = 1		// whether or not the race has sexual characteristics. at the moment this is only 0 for skeletons and shadows
 
-	var/face_y_offset = 0
-	var/hair_y_offset = 0
+	var/offset_features = list("uniform" = list(0,0), "id" = list(0,0), "gloves" = list(0,0), "glasses" = list(0,0), "ears" = list(0,0), "shoes" = list(0,0), "s_store" = list(0,0), "mask" = list(0,0), "head" = list(0,0),"face" = list(0,0), "belt" = list(0,0), "back" = list(0,0), "suit" = list(0,0), "neck" = list(0,0))
```

Comment:
Don't use magic strings.

Reply:
does that mean i should add it to the species on init or what